Products/Zing/send_organizers.py

```diff
@@ -0,0 +1,48 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2021 all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+from __future__ import absolute_import
+
+from Products.Jobber.task import requires, Abortable, DMD
+from Products.Jobber.zenjobs import app
+
+from .fact import device_organizer_info_fact
+from .interfaces import IZingConnectorProxy
+
+
+@app.task(
+    bind=True,
+    base=requires(Abortable, DMD),
+    name="zen.zenjobs.zing.send_organizers",
+    summary="Send organizer facts to Zing",
+    ignore_result=False,
+)
+def send_organizers(self):
+    facts = {}
+    for root_name in ("Devices", "Groups", "Locations"):
```

Comment:
Systems?

Reply:
Right.  I knew I forgot one.  This is what happens you only use zendmd and don't look in the UI.